app/controllers/users/email_confirmations_controller.rb

```diff
@@ -90,6 +90,17 @@ def message_for_already_confirmed_user
       end
     end
 
+    def flash_message_for_successful_signed_in_confirmation
+      if IdentityConfig.store.feature_select_email_to_share_enabled
+        t(
```

Comment:
Do we want this message about going to connected accounts to show for _every_ email confirmation? I would have expected it should only show when adding an email in the course of changing the email associated with a partner.

i.e.

1. Go to http://localhost:3000
2. Sign in
3. From account dashboard, add new email
4. Click "Confirm email address" from received email

Expected: Alert banner says "You have confirmed your email address"

Reply:
I was under the impression it was for every email confirmation so that if users add an email, they can update their email for associated accounts. For the scenario ur describing, I would think that they would be redirected back to the completion screen correct?

Reply:
I'm talking about a scenario where the user is logging directly into https://secure.login.gov, not through a partner.

Reply:
Looks like u were correct. I updated the code to reflect that change.